framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -378,15 +378,15 @@ int MediaPlayerImpl::getVolume()
 	return ret;
 }
 
-void MediaPlayerImpl::getVolumePlayer(int &ret)
+void MediaPlayerImpl::getVolumePlayer(uint8_t &ret)
```

Comment:
I think pointer is more meaningful instead of ref. please test & modify as below.
mpw.enQueue(&MediaPlayerImpl::getVolumePlayer, shared_from_this(), &ref);
void MediaPlayerImpl::getVolumePlayer(uint8_t *ret)

Reply:
Okay, but junmin suggests me to modify this in another PR and I think that is more proper.
How do you think?